lib/rspec/abq.rb

```diff
@@ -148,8 +148,20 @@ def self.setup_extensions_if_enabled!
       Extensions.setup!
     end
 
+    # Base ABQ error.
+    Error = Class.new(StandardError)
```

Comment:
just for my knowledge - is this equivalent to

```
class Error < StandardError
end
```

?

Reply:
Yep!